01-introduction.md

```diff
@@ -51,7 +51,7 @@ Much of the research in education has rightly focused on people without these pr
 While what we know about educational psychology can constrain teaching,
 it doesn't dictate it.
 In particular,
-there are often several plausible approaches to teaching
+there are often several possible approaches to teaching
```

Comment:
I prefer the original wording - "plausible" as in "intuitively appealing and internally consistent", which is stronger than "possible".